packages/web-components/src/components/content-item-row/content-item-row-media-featured.ts

```diff
@@ -25,16 +25,16 @@ const { prefix, stablePrefix: c4dPrefix } = settings;
 class C4DContentItemRowMediaFeatured extends C4DContentItem {
   render() {
     return html`
-      <div class="${prefix}--content-item-row__row">
-        <div class="${prefix}--content-item-row__col">
+      <div class="${prefix}--content-item-row__row" part="row-eyebrow">
```

Comment:
This is interesting. This same class is reused in multiple different contexts (here in `content-item-row-media-featured` as well as `content-item-row-media`). I would suggest the following, giving it a generic part name that is shared across all uses, as well as a modifier to that name to mark specific uses. Also to have the modifier mirror the component to reflect how the existing styles are scoped, eg.

```suggestion
      <div class="${prefix}--content-item-row__row" part="row row--media-featured">
```

Reply:
I see what you mean, but I'd argue this can already be solved with the following selectors:
```css
:host(dds-content-item-row-media)::part(row) { ... }
:host(dds-content-item-row-media-featured)::part(row) { ... }
```

I'd actually rather see something like this to differentiate the rows:
```suggestion
      <div class="${prefix}--content-item-row__row" part="row row--eyebrow">
```